portfolio/src/main/webapp/script-home.js

```diff
@@ -48,15 +48,24 @@ function main() {
         function adjustForwardOne() {
           slideShowGallery.adjustSlideManual(ADJUST_FORWARD); 
     }
+    // Fetches the pre-made comment list from the servlet and makes 
+    // a list for each comment.
+    fetch('/data').then(response => response.json()).then((comments) => {
+      const /** ?HTMLCollection */commentContainer =
+          document.getElementById('comments-container');
+      commentContainer.innerHTML = '';
+      for(i in comments){
```

Comment:
for-each loops dont use i and always use a scoped variable.
for (i in comments) should be for (const comment : comments)

Reply:
I was trying to do this earlier but I got a message saying that "Type annotations can only be used in TypeScript files"; I am not sure what that means.